### PR TITLE
Update fr.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**

### CODE REVIEW

**Review by UltraHDR — DISMISSED** [review]

File: streams/fr.m3u [hunk 1]
```diff
@@ -84,7 +84,7 @@ https://mn-nl.mncdn.com/awraastv/awraastv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="FashionTVCzechSlovak.fr",FashionTV Czech&Slovak (450p) [Not 24/7]
 http://lb.streaming.sk/fashiontv/stream/playlist.m3u8
 #EXTINF:-1 tvg-id="Franceinfo.fr",Franceinfo (720p)
-https://raw.githubusercontent.com/pshanmu3/dailymotion_to_m3u8/main/FranceInfo.m3u8
```
Comment (UltraHDR) [inline streams/fr.m3u:87]: works

File: streams/fr.m3u [hunk 2]
```diff
@@ -65,8 +65,8 @@ https://ncdn-live-bfm.pfd.sfr.net/shls/LIVE$BFM_TOULONVAR/index.m3u8?end=END&sta
 https://biptv.tv/live/biptvstream_orig/index.m3u8
 #EXTINF:-1 tvg-id="BoomerangFrance.fr",Boomerang France (720p)
 http://51.81.185.2/BOOMERANG/index.m3u8
-#EXTINF:-1 tvg-id="Canal32.fr",Canal 32 (480p)
-https://edge13.vedge.infomaniak.com/livecast/ik:canal32/manifest.m3u8
```
Comment (UltraHDR) [inline streams/fr.m3u:69]: works

File: streams/fr.m3u [hunk 3]
```diff
@@ -8,7 +8,7 @@ https://video1.getstreamhosting.com:1936/8420/8420/playlist.m3u8
 #EXTINF:-1 tvg-id="ADNTVPlus.fr",ADN TV+ (1080p) [Geo-Blocked]
 https://samsunguk-adn-samsung-fre-qfrlc.amagi.tv/playlist/samsunguk-adn-samsung-fre/playlist.m3u8
 #EXTINF:-1 tvg-id="Africa24.fr",Africa 24 (540p)
-https://edge12.vedge.infomaniak.com/livecast/ik:africa24/manifest.m3u8
```
Comment (UltraHDR) [inline streams/fr.m3u:11]: works

File: streams/fr.m3u [hunk 4]
```diff
@@ -84,7 +84,7 @@ https://mn-nl.mncdn.com/awraastv/awraastv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="FashionTVCzechSlovak.fr",FashionTV Czech&Slovak (450p) [Not 24/7]
 http://lb.streaming.sk/fashiontv/stream/playlist.m3u8
 #EXTINF:-1 tvg-id="Franceinfo.fr",Franceinfo (720p)
-https://raw.githubusercontent.com/pshanmu3/dailymotion_to_m3u8/main/FranceInfo.m3u8
+https://ythls.onrender.com/channel/UCO6K_kkdP-lnSCiO3tPx7WA.m3u8
```
Comment (UltraHDR) [inline streams/fr.m3u:87]: Doesn't work

File: streams/fr.m3u [hunk 5]
```diff
@@ -221,3 +223,69 @@ http://livevideo.vedge.infomaniak.com/livecast/ik:bblackclassik/manifest.m3u8
 https://ythls.onrender.com/channel/UCMkQQZsNdRVZ3N7Oj6jVnyg.m3u8
 #EXTINF:-1 tvg-id="4KurdHD.fr",4Kurd HD (720p) [Not 24/7]
 http://51.210.199.23/hls/stream.m3u8
+#EXTINF:-1 tvg-id="",Pearl Diffusion  (720p) [Not 24/7]
```
Comment (UltraHDR) [inline streams/fr.m3u:226]:
```suggestion
#EXTINF:-1 tvg-id="",Pearl Diffusion (720p) [Not 24/7]
```

**Comment by solo489** [comment]
Update jp.m3u

File: streams/fr.m3u (hunk 4 above)
Comment (Sphinxroot) [reply streams/fr.m3u:87]: geo stream ?

File: streams/fr.m3u [hunk 6]
```diff
@@ -161,6 +161,8 @@ https://ott.tv5monde.com/Content/HLS/Live/channel(pacifique)/variant.m3u8
 https://ott.tv5monde.com/Content/HLS/Live/channel(style1)/variant.m3u8
 #EXTINF:-1 tvg-id="TV7Bordeaux.fr",TV7 Bordeaux (720p)
 https://live.digiteka.com/1/RVNZMmJtUURmd09I/U0dNU0tk/hls/live/playlist.m3u8
+#EXTINF:-1 tvg-id="TV7Bordeaux.fr",TV7 Bordeaux (720p)
+https://ythls.onrender.com/channel/UCV9yrdAfyd1BdDyd4tDdVxg.m3u8
```
Comment (AadithyanRaju) [inline streams/fr.m3u:165]: works

File: streams/fr.m3u [hunk 7]
```diff
@@ -221,3 +223,69 @@ http://livevideo.vedge.infomaniak.com/livecast/ik:bblackclassik/manifest.m3u8
 https://ythls.onrender.com/channel/UCMkQQZsNdRVZ3N7Oj6jVnyg.m3u8
 #EXTINF:-1 tvg-id="4KurdHD.fr",4Kurd HD (720p) [Not 24/7]
 http://51.210.199.23/hls/stream.m3u8
+#EXTINF:-1 tvg-id="",Pearl Diffusion (720p) [Not 24/7]
+https://ythls.onrender.com/channel/UCvpCjGQASCazC-Vxcp9tgPA.m3u8
```
Comment (AadithyanRaju) [inline streams/fr.m3u:227]: Doesn't work

**Comment by freearhey** [comment]
Closing due to lack of activity.

**Pull request closed without merging**